Algorithms-BubbleSort.md

```diff
@@ -0,0 +1,51 @@
+#BubbleSort
+
+Bubble sort is a simple sorting algorithm. This sorting algorithm is comparison based algorithm in which each pair of adjacent elements is
```

Comment:
Again, keep in mind that your readers are not necessarily from a computer science background. Start with simpler language about what sorting is, why this is important.

I think for now, no need to changing anything in this document. Later on, we would create a master document, like [Python-Functions](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Python-Functions) that explains what sorting is, why it is useful to learn and link to all sorting articles.